lightning/src/ln/channelmanager.rs

```diff
@@ -9357,6 +9347,7 @@ where
 	/// [`MessageRouter::create_compact_blinded_paths`].
 	///
 	/// Errors if the `MessageRouter` errors.
+	#[allow(unused)]
 	fn create_compact_blinded_paths(&self, context: OffersContext) -> Result<Vec<BlindedMessagePath>, ()> {
```

Comment:
Why not remove this instead?

Reply:
Remove the function instead of using #[allow(unused)] in [pr3246.06](https://github.com/shaavan/rust-lightning/commits/pr3246.06)
Thanks for the pointer!